src/Benchmarks/Startup.cs

```diff
@@ -49,7 +51,7 @@ public void ConfigureServices(IServiceCollection services)
             services.AddSingleton<ApplicationDbSeeder>();
             services.AddEntityFrameworkSqlServer()
```

Comment:
If you are not using the application service provider for internal EF services, then there is no point in adding all these services, so you should remove AddEntityFrameworkSqlServer.